src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -16,7 +16,9 @@
 using Azure.Functions.Cli.Interfaces;
 using Azure.Functions.Cli.StacksApi;
 using Colors.Net;
+using Dynamitey;
```

Comment:
Why do we need this?

Reply:
Probably auto added. Let me try to remove it.